examples/spring_boot/src/main/java/hello/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+java_library(
+    name = "lib",
+    srcs = glob(["*.java"]),
+    deps = [
+        "@maven//:org_springframework_boot_spring_boot_starter_web",
```

Comment:
You only specified this one. I think it's a bad practice to not specify all direct deps used and relying on rules_jvm_external to pull direct deps in from a different direct dep. Could you add the other deps to maven_install?

Reply:
Done.